Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -224,9 +242,32 @@ def examineFiles(self, files):
             for tag in subseriesTags:
                 value = slicer.dicomDatabase.fileValue(file, self.tags[tag])
                 value = value.replace(",", "_")  # remove commas so it can be used as an index
+
                 if tag not in subseriesValues:
                     subseriesValues[tag] = []
-                if value not in subseriesValues[tag]:
+
+                if tag in vectorTags:
+                    if value != "":
+                        vector = self.tagValueToVector(value)
+
+                        found = False
+                        for subseriesValue in subseriesValues[tag]:
+                            subseriesVector = self.tagValueToVector(subseriesValue)
+
+                            # The 'self.orientationEpsilon' value default is set to 1.e-6, which reproduces the same behavior as
```

Comment:
Movr this to the top where we define the ne the 1e-6 value

Reply:
done in https://github.com/Slicer/Slicer/pull/7362/commits/3c2da06844dc014a62795d35e14c8787fd0dfa42